src/Utility.ts

```diff
@@ -40,6 +41,17 @@ export namespace Utility {
         });
     }
 
+    export function trackTelemetryStep(step: string): void {
+        const session: Session = TelemetryWrapper.currentSession();
+        if (session && session.extraProperties) { session.extraProperties.finishedSteps.push(step); }
```

Comment:
where is finishedSteps used?

Reply:
TelemetryWrapper track all finished steps in command end event.